tests/integration_tests/with_db/udfs/test_token_classification_script.py

```diff
@@ -37,20 +50,58 @@ def test_token_classification_script(
 
     # execute sequence classification UDF
     result = db_conn.execute(query).fetchall()
-
     # assertions
     assert result[0][-1] is None
-    added_columns = 6  # start_pos,end_pos,word,entity,score,error_message
-    removed_columns = 1  # device_id
-    n_cols_result = len(input_data[0]) + (added_columns - removed_columns)
-    assert len(result) >= n_rows and len(result[0]) == n_cols_result
+    # added_columns: start_pos,end_pos,word,entity,score,error_message
+    # removed_columns: device_id
+    assert correct_number_of_results(6, 1,
```

Comment:
correct_number_of_results should be probably also assert_correct_number_of_results and assert directly the condition + maybe a good assert message

Reply:
changed